updatecli.d/default.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: "Helm autodiscovery using git scm"
+
+autodiscovery:
+  crawlers:
+    helm:
+      ignore:
+        - path: charts/alfresco-search-service/*
```

Comment:
what's the problem with search service?

at least describe it in the pr body or inline comments in one of the updatecli yaml